scripts/modules/models-checkout.js

```diff
@@ -344,7 +344,8 @@
                 var me = this;
                 this.isLoading(true);
                 var order = this.getOrder();
-                if (order) {
+                var isVisaCheckout = this.get('paymentWorkflow') === 'VisaCheckout';
```

Comment:
why are you checking for `!isVisaCheckout`? what if it is a regular visa checkout, not from quote?

Reply:
Removed the `isVisaCheckout` check from the condition.